source/main.cpp

```diff
@@ -143,13 +145,25 @@ void onNewBatteryData(MicroBitEvent)
     uart->send(buffer, ASYNC);
 }
 
+void onNewLightSensorsData(MicroBitEvent)
+{
+    if (connected == 0)
+    {
+        return;
+    }
+    uBit.sleep(1);  // Prevents an 020 error. 🤷
+    char buffer[23];
+    snprintf(buffer, 23, "light-sens:%d,%d", lightSensors.getLeftReading(), lightSensors.getRightReading());
+    uart->send(buffer, ASYNC);
+}
+
 
 int main()
 {
     // Initialise the micro:bit runtime.
     uBit.init();
 
-    // uBit.accelerometer.setPeriod(1000);  // In case you want to slow it down. Defaults to 20ms.
+    uBit.accelerometer.setPeriod(1000);  // milliseconds

```

Comment:
Do we want to have the accelerometer run this slowly?

Reply:
It seemed to be causing me pretty frequent 020 errors when it ran full speed. Maybe we can play with this number alongside all the others when we have everything in?